desktop/libs/notebook/src/notebook/templates/editor_components.mako

```diff
@@ -2051,10 +2097,11 @@ ${ hueIcons.symbols() }
       var snip = viewModel.selectedNotebook().snippets()[0];
       if (snip) {
         snip.statement_raw(content);
-        snip.ace().setValue(content);
+        snip.ace().setValue(content, 1);
       }
       hideHoverMsg(viewModel);
     }
+    window.replaceAce = replaceAce;
```

Comment:
Yes, I have to call it to replace the content of the notebook here: https://github.com/cloudera/hue/pull/342/commits/2e2efed50b37c2a575f21c6178fa070f682e18cc#diff-7ac1d1c7d51666daaf236286c4edb3c8R64